src/tool/parseSVG.ts

```diff
@@ -627,9 +628,19 @@ function parseGradientColorStops(xmlNode: SVGElement, gradient: GradientObject):
             // <stop stop-color="red"/>
             const styleVals = {} as Dictionary<string>;
             parseInlineStyle(stop, styleVals, styleVals);
-            const stopColor = styleVals.stopColor
+            let stopColor = styleVals.stopColor
                 || stop.getAttribute('stop-color')
                 || '#000000';
+            const stopOpacity = styleVals.stopOpacity
+                || stop.getAttribute('stop-opacity');
+            if (stopOpacity) {
```

Comment:
Should use `stopOpacity != null` to avoid `0` being ignored here.

Reply:
Both getAttribute and styleVars are always string type and the string '0' is truthy.